quantstats/stats.py

```diff
@@ -252,7 +252,7 @@ def adjusted_sortino(returns, rf=0, periods=252, annualize=True):
     direct comparisons to the Sharpe. See here for more info:
     https://archive.is/wip/2rwFW
     """
-    data = sortino(returns, rf=0, periods=periods, annualize=annualize)
+    data = sortino(returns, rf=0, periods=periods, annualize=True)
```

Comment:
should not always pass `True` to annualize but take argument from method call (keep previous version)